.vscode/settings.json

```diff
@@ -43,6 +43,6 @@
   "typescript.surveys.enabled": false,
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
**question:** is this something related to the eslint version in metaphysics?

Reply:
it is already "explicit" probably my branch was outdated, after a rebase that was solved 👍